docs: add demo gallery

Mini256 opened the pull request:
<img width="1398" height="952" alt="image" src="https://github.com/user-attachments/assets/9a6ab7cf-7402-4bed-b3db-7238bb3a39c5" />

xuechunL commented on layouts/community-cn/organization.html:
```diff
@@ -96,6 +96,20 @@
+                      </div>
+                      <div class="c-member">
+                        <div class="avatar">
+                          <img src="/images/community/organization/Jun-Seok Heo.jpeg">
```
Please format the src path for the image.

Mini256 committed:
feat: migrate dependencies to pyproject.toml and add dependency checks

- Consolidate all dependencies into pyproject.toml for better management
- Remove separate scripts/requirements.txt file
- Update Makefile to use 'pip install -e .' for installation
- Add dependency check script (scripts/check_dependencies.py)
- Update documentation to reflect new dependency management
- Add 'make check' command to verify environment setup
- Fix Examples ordering in navigation and sync script
- All dependencies now managed in single pyproject.toml file

Mini256 committed:
feat: migrate to UV package manager for faster dependency management

- Update Makefile to use 'uv pip install -e .' instead of pip
- Add UV availability check in scripts/check_dependencies.py
- Update documentation to include UV prerequisites
- Add UV installation instructions in README.md and docs/
- Enhance Makefile help with UV information
- Add comments in pyproject.toml about UV usage
- UV provides significantly faster package installation and resolution

UV installation: curl -LsSf https://astral.sh/uv/install.sh | sh

Mini256 committed:
simplify: streamline dependency check script

- Reduce verbose checks to just essential ones
- Only check UV availability and core imports
- Much faster and cleaner output
- Focus on what actually matters for getting started

Mini256 committed:
fix

Mini256 committed:
feat: add example gallery and enhance example documentation

Mini256 committed:
refine generate-demos script

Mini256 committed:
update deps

Mini256 committed:
refine
